File: palindrome.py
def palindrome(is_palindrome):
    reverse_string = ""  
    size_palindrome = len(is_palindrome) - 1  
    
    while size_palindrome >= 0:
        reverse_string = reverse_string + is_palindrome[size_palindrome]
        size_palindrome = size_palindrome-1
    if reverse_string == is_palindrome:
        print("Is Palindrome")
    else:
        print("NOT a Palindrome")
        
        
s = "radars"
palindrome(s)

File: Palindrome.java
public class Palindrome {

    public static void main(String[] args) {
        String isPalindrome = "accbcca";
        String reverseString = "";

        int indexPalindrome = isPalindrome.length() - 1;

        for (int i = indexPalindrome; i >= 0; i--) {
            reverseString = reverseString + isPalindrome.charAt(i);
        }

        if (reverseString.toLowerCase().equals(isPalindrome.toLowerCase())) {
            System.out.println("Is Palidrome");
        } else
            System.out.println("NOT PAlindrome");

    }

}
